src/views/main-page/basic-information/MainPageBasicInformation.scss

```diff
@@ -0,0 +1,32 @@
+@import 'styles/palette';
+@import 'styles/mixins';
```

Comment:
Mixins will be enough. Mixins already imports palette so this palette import is not needed